### PR TITLE
Wire scenes with Redux placeholders

### PR DESCRIPTION
**Pull request by gmarty**
This PR prepares the scene components to work with Redux.
It's only placeholders for now, but then in the future we can easily extract data from the state and actions to place in these components props.

### CODE REVIEW

File: js/scenes/Item.js [hunk 1]
```diff
@@ -53,3 +54,16 @@ const styles = StyleSheet.create({
...
+};
+
+const mapDispatchToProps = () => {
+  return {};
+};
+
+export default connect(
+  mapStateToProps,
+  mapDispatchToProps
+)(Item);
```
Comment (wilsonpage) [inline js/scenes/Item.js:69]: I don't think you need to export a 'raw' and 'reduxed' version of the component for testing. When you're testing I think you can just provide a mock `store` property to the `connect()`ed component and it should 'just work' :)

Comment (gmarty) [reply js/scenes/Item.js:69]: I followed the recommendations of [Redux about testing](http://redux.js.org/docs/recipes/WritingTests.html). I tried first the approach you suggested but it means that any components (bet it scenes or minor UI elements) must be provided with a store in every test file. I found it ended up adding too much code.

Comment (wilsonpage) [reply js/scenes/Item.js:69]: Oh cool OK, ignore me then :p

**Comment by arcturus** [comment]
shiiiip it

**Pull request merged**